packages/syntax-core/src/Card/Card.tsx

```diff
@@ -18,7 +18,7 @@ type CardType = {
    * `sm`: 352px
    * `lg`: 744px
    *
-   * @defaultValue sm
+   * @defaultValue `undefined`
```

Comment:
@zprong pending slack discussion but could we add `@deprecated` to `size`?

Just make sure we do update the places in our code where we do use `<Card />` without a `size` prop at the moment

Reply:
^ For posterity, we'll just make the changes to replace all instances of `size` with `maxWidth` instead of marking as `@deprecated`